web/beacon-app/src/features/home/components/QuickStart.tsx

```diff
@@ -1,9 +1,12 @@
+import OtterLookingDown from '@/components/icons/otter-looking-down';
+
 import SetupNewProject from '../components/SetupNewProject';
 import AccessDocumentationStep from './AccessDocumentationStep';
 
 export default function QuickStart() {
   return (
-    <div className="space-y-10">
+    <div className="relative w-full space-y-10">
```

Comment:
`Relative` and `w-full` were added to make sure the image doesn't move too much when a user adjusts their screen width.